docs_user/modules/proc_adopting-networker-services-to-the-data-plane.adoc

```diff
@@ -305,39 +291,37 @@ spec:
   - openstack-networker
 EOF
 ----
-
-Note: alternatively, you could include the `Networker` specific node set in the
-`nodeSets` list when creating the main, `openstack`,
-`OpenStackDataPlaneDeployment` CR.
-
-Note that once a `OpenStackDataPlaneDeployment` CR is deployed, it's impossible
-to add new node sets to it later.
++
+[NOTE]
+Alternatively, you can include the `Networker` specific node set in the
+`nodeSets` list when you create the main `OpenStackDataPlaneDeployment` CR, before you deploy it. Note that once an `OpenStackDataPlaneDeployment` CR is deployed, you cannot add new node sets to it later.
 
 .Verification
 
 . Confirm that all the Ansible EE pods reach a `Completed` status:
 +
 ----
 # watching the pods
-watch oc get pod -l app=openstackansibleee
+$ watch oc get pod -l app=openstackansibleee
 ----
 +
 ----
 # following the ansible logs with:
-oc logs -l app=openstackansibleee -f --max-log-requests 20
+$ oc logs -l app=openstackansibleee -f --max-log-requests 20
 ----
 
 . Wait for the data plane node set to reach the `Ready` status:
 +
 ----
-oc wait --for condition=Ready osdpns/openstack-networker --timeout=30m
+$ oc wait --for condition=Ready osdpns/<networker_CR_name> --timeout=30m
 ----
++
+* Replace `<networker_CR_name>` with the name of the CR that you deployed for your Networker nodes, for example, `openstack-networker`.
 
-. Verify that {networking_service} agents are alive (the list of agents may
-vary depending on the services you've enabled):
+. Verify that the {networking_first_ref} agents are running. The list of agents might vary depending on the services you enabled:
```

Comment:
Would you say the list "might" vary depending on the services enabled or it definitely will depend on the services enabled?
Maybe:
"The list of agents depends on the services you enabled:"
or 
"The list of agents differs depending on the services you enabled:"
or 
"The services that you enabled determines the list of agents:"
or 
""The list of agents displays the services that you enabled."
or
"The list of agents displays the services that you enabled and might differ from your environment:"